packages/hawtio/src/plugins/shared/chart/WatchableAttributesForm.tsx

```diff
@@ -67,17 +73,19 @@ export const WatchableAttributesForm = ({
           ) => {
             const newWatches: AttributesToWatch = {}
             newWatchedAttributes.forEach(newWatchedAttribute => {
-              newWatches[newWatchedAttribute.id] = {
-                ...Object.fromEntries(newWatchedAttribute.children?.map(({ id }) => [id, true]) || []),
+              newWatches[newWatchedAttribute.text] = {
+                ...Object.fromEntries(newWatchedAttribute.children?.map(({ text }) => [text, true]) || []),
               }
             })
             newUnwatchedAttributes.forEach(newUnwatchedAttribute => {
-              newWatches[newUnwatchedAttribute.id] = {
-                ...(newWatches[newUnwatchedAttribute.id] || {}),
-                ...Object.fromEntries(newUnwatchedAttribute.children?.map(({ id }) => [id, false]) || []),
+              newWatches[newUnwatchedAttribute.text] = {
+                ...(newWatches[newUnwatchedAttribute.text] || {}),
+                ...Object.fromEntries(newUnwatchedAttribute.children?.map(({ text }) => [text, false]) || []),
               }
             })
 
+            console.log(newWatches)
```

Comment:
don't leave `console.log`. if you want to leave some debug logging, use `log.debug` from `globals.ts`

Reply:
So sorry. Removing this right away.

I'm going to build a custom lint rule at this rate so it tells me of any console.logs. It happens too much